### PR TITLE
fill pdf with data

### LINKED ISSUE #278: fill fields in PDF

### PR DESCRIPTION
**Pull request by crapStone**
closes #278

### CODE REVIEW

File: src-tauri/cabr2_load_save/src/pdf/mod.rs [hunk 1]
```diff
@@ -68,17 +70,19 @@ impl Saver for PDF {
...
     stylesheet: &'a String,
-    document: &'a CaBr2Document,
+    document: &'a PDFCaBr2Document,
   }
 
   lazy_static! {
     static ref REG: Arc<Mutex<Option<(String, Handlebars<'static>)>>> = Arc::new(Mutex::new(None));
   }
 
+  log::debug!("converted data: {:#?}", document);
```
Comment (6543) [inline src-tauri/cabr2_load_save/src/pdf/mod.rs:84]: trace

Comment (crapStone) [reply src-tauri/cabr2_load_save/src/pdf/mod.rs:84]: delete

**Comment by crapStone** [comment]
missing pieces:

- [ ] ghs symbols
- [ ] small "cosmetic" fixes

File: src-tauri/cabr2_load_save/src/pdf/mod.rs [hunk 2]
```diff
@@ -68,27 +70,30 @@ impl Saver for PDF {
...
+
   let mut reg = REG.lock().unwrap();
 
-  if reg.is_none() {
+  if true {
```
Comment (6543) [inline src-tauri/cabr2_load_save/src/pdf/mod.rs:88]: ?

File: src-tauri/cabr2_load_save/src/pdf/mod.rs [hunk 3]
```diff
@@ -68,27 +70,30 @@ impl Saver for PDF {
...
-    document: &'a CaBr2Document,
+    document: &'a PDFCaBr2Document,
   }
 
   lazy_static! {
     static ref REG: Arc<Mutex<Option<(String, Handlebars<'static>)>>> = Arc::new(Mutex::new(None));
   }
 
+  log::debug!("converted data: {:#?}", document);
+
```
Comment (6543) [inline src-tauri/cabr2_load_save/src/pdf/mod.rs:85]:
```suggestion
```

**Review by 6543 — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Review by Epsilon02 — APPROVED** [review]

**Pull request merged**